docs/slides.md

```diff
@@ -29,7 +29,48 @@ Carl Pacey and Mark Tse
 
 ---
 
-# Exercise 2: Implementing a feature using small commits
+# Exercise 2: Let's add a new feature
+
+* You're hopefully familiar with Users after Exercise #1.
+* You may have noticed that the app already supports `/fizzbuzz/{number}`.
+* Now we're going to expand the `/fizzbuzz` functionality to include user information.
+
+---
+
+# Exercise 2: Best route ever
+
+We want to add the following route:
+
+`/fizzbuzz/{number}/users/{userId}`
+
+This route will return a modified FizzBuzz:
+
+1. Instead of "Fizz", this route will use the user's UserName.
+1. Instead of "Buzz", this route will use a new user property, BuzzWord.
+
+---
+
+# Exercise 2: Specification by Example
+
+```http
+GET /users/1045
+```
+
+```json
+{
+  "id": 1045,
+  "userName": "cpacey",
+  "isActive": true,
+  "buzzWord": "isawesome"
+}
+```
+
+| `GET` |      Result      |
+|:----------|:-------------|
+| `/fizzbuzz/14/users/1045` |  `29` |
+| `/fizzbuzz/30/users/1045` | `cpacey` |

```

Comment:
30 is divisible by 5, and technically should return `cpaceyisawesome`.

Reply:
Wow I suck.